service/dap/server.go

```diff
@@ -1411,6 +1411,12 @@ func (s *Server) onConfigurationDoneRequest(request *dap.ConfigurationDoneReques
 		}
 		s.send(e)
 	}
+	func() {
```

Comment:
It is not necessary to lock the target for this here, I think.

Reply:
I see you've approved this with this comment outstanding. Would you like this changed? Can anything ever race for this?

Reply:
Yes, I still think it's unnecessary.

Reply:
There shouldn't be an issue with adjusting this without locking, so I'll remove the lock.